pyam/core.py

```diff
@@ -117,6 +117,9 @@ def __getattr__(self, attr):
 
         return super().__getattr__(attr)
 
+    def __len__(self):
+        return sum(self.values)
```

Comment:
You should always prefer the pandas native methods.

```suggestion
        return self.sum()
```